scripts/corelli/calibration/database.py

```diff
@@ -59,6 +59,35 @@ def filename_bank_table( bankID:int, database_path:str, table_type:str = 'calibr
     return filename
 
 
+def save_manifest_file( bankIDs:list, database_path:str)->None:
+    """
+    Function that saves or updates an existing manifest_corelli_date.csv file.
+    There is one file stored for each calibration day. If file exist it will append two columns:
+    bankID, timestamp (ISO format)
+    :param bankIDs input of bankIDs that have been calibrated
+    :param database_path location of the corelli database (absolute or relative)
+           Example: database/corelli/ for manifest file:
+                    database/corelli/manifest_corelli_YYYYMMDD.csv
+    """
+
+    time:str = datetime.now().strftime('%Y%m%d') # format YYYYMMDD
```

Comment:
I think this is not the correct day-stamp that should be assigned to the input bank ID's. In general, each bank calibration file that ends up being retrieved when compiling the full-instrument calibrated file has its own day-stamp. The day-stamp for a particular bank should be the day-stamp of the events file that was used to calibrate that bank.

Reply:
Sounds good, thanks @jmborr . I will accommodate and allow for an input date on the save functions.

Reply:
As I understand your code, if you're passing an array of bank ID's to `save_manifest_file()`, you could also pass an array of same length containing the day-stamp for each bank ID.